actions.go

```diff
@@ -76,6 +78,12 @@ func (s *shellActionsImpl) ReadLine() string {
 	return line
 }
 
+func (s *shellActionsImpl) ReadLineWithDefault(defaultValue string) string {
+	s.reader.defaultValue = defaultValue
+	line, _ := s.readLine()
```

Comment:
You need to reset `defaultValue` back to empty after calling `s.readLine()`.
```suggestion
	line, _ := s.readLine()
	s.reader.defaultValue = ""
```